### ISSUE TITLE
React Native apps fail to build since NDK update to r17 (due to removed support for ARMv5/armeabi)

### ISSUE THREAD

**Issue by sunweiyang** [facebook/react-native#19321]
According to release notes for NDK r17, which was released a couple days ago (https://developer.android.com/ndk/downloads/revision_history):
```
Support for ARMv5 (armeabi), MIPS, and MIPS64 has been removed. Attempting to build any of these ABIs will result in an error.
```
It seems like React Native still references `armeabi-v7a` across its codebase. 

In the meantime, I had to use a workaround (explained in original issue at android-ndk/ndk#700) to get my React Native projects built for Android. (A true solution would remove all references to `armeabi-v7a` from the codebase, but this is something quick-and-dirty for the lazy or time-constrained.)

EDIT: For everyone's convenience, the workaround (macOS) is to replace the bare NDK r17 `~/Library/Android/sdk/ndk-bundle/toolchains/mips64el-linux-android-4.9/` and `~/Library/Android/sdk/ndk-bundle/toolchains/mipsel-linux-android-4.9/` directories with those of the old NDK r16.

**Comment by sunweiyang** [comment]
I believe the issue template isn't quite relevant to this issue, which discusses Android NDK's relationship to the RN codebase as a whole. Maintainers, please let me know if this is not the case, and I'll gladly provide any other information you wish.

**Comment by hramos** [comment]
I agree, it's OK to ignore the template in this particular case. The bot shouldn't nag again now that the For Discussion label has been applied.

**Comment by steverob** [comment]
Any updates on this?

**Comment by krystiansliwa** [comment]
Any workaround for that?

**Comment by kelset** [comment]
Please don't pollute this issue with "me too/any updates". 

Until better solution:
1) Don't update the NDK unless strictly necessary
2) follow the workaround OP linked

**Comment by sunweiyang** [comment]
For everyone's convenience, the workaround (macOS) is to replace the bare NDK r17 `~/Library/Android/sdk/ndk-bundle/toolchains/mips64el-linux-android-4.9/` and `~/Library/Android/sdk/ndk-bundle/toolchains/mipsel-linux-android-4.9/` directories with those of the old NDK r16.

**Comment by LegNeato** [comment]
This is likely relevant here: https://github.com/facebook/buck/pull/1894

**Comment by gengjiawen** [comment]
Is there some misunderstanding here, react native support `armeabi-v7a`, it's not `armeabi`. But those two compatible, I don't see ndk r17 is problem here.

**Comment by dulmandakh** [comment]
Closing this issue because RN landed Android NDK r17b and builds just fine (https://github.com/facebook/react-native/pull/20357). And will be released in 0.57.
